internal/provider/cluster_data_source.go

```diff
@@ -33,68 +33,84 @@ func (d *clusterDataSource) Schema(
 	_ context.Context, _ datasource.SchemaRequest, resp *datasource.SchemaResponse,
 ) {
 	resp.Schema = schema.Schema{
+		Description: "CockroachDB Cloud cluster. Can be dedicated or serverless.",
 		Attributes: map[string]schema.Attribute{
 			"id": schema.StringAttribute{
 				Required: true,
 			},
 			"name": schema.StringAttribute{
-				MarkdownDescription: "Name of cluster",
+				MarkdownDescription: "Name of the cluster.",
 				Computed:            true,
 			},
 			"cockroach_version": schema.StringAttribute{
-				Computed: true,
+				Computed:    true,
+				Description: "Full version of CockroachDB running on the cluster.",
 			},
 			"plan": schema.StringAttribute{
-				Computed: true,
+				Computed:    true,
+				Description: "Denotes what type of plan the cluster is on, e.g. 'DEDICATED' or 'SERVERLESS'.",
 			},
 			"cloud_provider": schema.StringAttribute{
 				Computed: true,
+				Description: "Cloud provider used to host the cluster. Allowed values are:" +
```

Comment:
supernit: data-sources/cluster.md looks unaffected by this, but should `MarkdownDescription` be used when the description contains a list?